lib/conta_azul_api/product.rb

```diff
@@ -10,20 +10,20 @@ class NotFound < StandardError; end
     MAX_PRODUCTS_PER_PAGE = 200
 
     def self.find(product_id)
-      product = ContaAzulApi::Request.new.get(
+      product_response = ContaAzulApi::Request.new.get(
```

Comment:
The implementation is amazing! Just thinking about the `Customer` class, could we adapt it to use `Request` model as well?

Reply:
Done in [f269a78](https://github.com/paulo-silva/conta_azul_api/pull/14/commits/f269a787d95f4736a49136c0902dbdb13695d974). 😄